ui/views.py

```diff
@@ -115,7 +137,10 @@ def get_context_data(self, **kwargs):  # pylint: disable=arguments-differ
     def get(self, request, *args, **kwargs):
         collection_key = kwargs['collection_key']
         if collection_key:
-            get_object_or_404(Collection, key=collection_key)
+            collection = get_object_or_404(Collection, key=collection_key)
```

Comment:
I think this can be simplified to:
```python
if collection_key:
    get_object_or_404(Collection, key=collection_key)
    return super().get(request, *args, **kwargs)
```